test/app.spec.js

```diff
@@ -0,0 +1,187 @@
+'use strict'
+
+const request = require('supertest')
+const chai = require('chai')
+const expect = chai.expect
+const mongoose = require('mongoose')
+const app = require('../src/app')
+
+describe('app', () => {
+  let id
+  let createdAt
```

Comment:
Can you please explain the reasoning for declaring `createdAt` here and with let?

Reply:
Prior to Tomasz comment about letting Mongoose handle the timestamps for updatedAt and createdAt, I captured it from the response so I could test later that it would not be improperly modified by an update (i.e. PUT). I made it a let here so that I can assign the value later but placed in a higher scope so it could be accessed by the test. That test was not in place but I have just added it. After setting timestamps equal to true in the mongoose schema, I am seeing behavior where the createdAt is being set to the same value as updatedAt. I am leaving this failing test in place so I take a look at it this weekend and troubleshoot what is going on. If you have any thoughts, any help would be appreciated :)